DataAccess/DataAccess.Test/PostgreModelsTest.cpp

```diff
@@ -0,0 +1,147 @@
+#include <catch2/catch.hpp>
+
+#include <Models/Models.hpp>
+#include <Models/ModelFactory.hpp>
+#include "DataAccess.Postgre/PostgreAdapter.hpp"
+#include "DataAccess.Postgre/DataBaseOptions.hpp"
+#include "DataAccess.Postgre/PostgreRepositories.hpp"
+#include "PostgreModel.hpp"
+#include <Models/UnifyedModel.hpp>
+
+
+using Models::User;
+using Models::UserInfo;
+using Models::ModelFactory;
+
+using Models::Channel;
+using Models::ChannelData;
+
+using DataAccess::PostgreModel;
+
+using DataAccess::PostgreAdapter;
+using DataAccess::PGQueryBuilder;
+
+
+TEST_CASE("Check model usage", "[dummy]")
+{
+    auto testTable = std::make_unique<PGQueryBuilder>("users", PostgreAdapter::Instance(DBOptions::test));
+
+    SECTION("Wanna add a user", "Kinda a 'userRegistration method in repos'")
+    {
+        User<PostgreModel> testUser({ {UserInfo::ID,"3"},
+                                    {UserInfo::EMAIL,"aboba3@gmail.com"},
+                                    {UserInfo::LOGIN,"aboba3"},
+                                    {UserInfo::PASSHASH,"kindahash3"} });
+
+        std::tuple insertData{std::pair{testUser.fieldName(UserInfo::ID), testUser[UserInfo::ID]},
+                              std::pair{testUser.fieldName(UserInfo::EMAIL), testUser[UserInfo::EMAIL]},
+                              std::pair{testUser.fieldName(UserInfo::LOGIN), testUser[UserInfo::LOGIN]},
+                              std::pair{testUser.fieldName(UserInfo::PASSHASH), testUser[UserInfo::PASSHASH]}};
+
+        auto responce = testTable->Insert()->columns(insertData)->execute();
+
+        REQUIRE_FALSE(responce.has_value());
+
+        SECTION("Check const with mutable")
```

Comment:
I don't understand how constancy can be checked through mutable if mutable contains const properties

Reply:
I wrote it mostly to myself to check how properly 'mutable' works with const objects but I guess it was written a couple of days ago and I reworked all tests due to new structure, so this test rn is useless, going to be reworked again

Reply:
I've deleted 'em, so I resolve